db/migrate/20160129090541_remove_type_from_lists.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveTypeFromLists < ActiveRecord::Migration[5.0]
+  def change
+    remove_column :lists, :type, :string
```

Comment:
Ah good call.  You may have discovered.. but the column named `type` has a special meaning in ActiveRecord and would have caused some odd errors.  There is a warning about this in the [Active Record Basics Guide](http://guides.rubyonrails.org/active_record_basics.html#schema-conventions)